src/components/ChatRoom.js

```diff
@@ -101,6 +120,7 @@ export default function ChatRoom() {
               <td>{chatroom.name}</td>
               <td>{chatroom.members.map((member) => users[member]).join(", ")}</td>
               <td>{users[chatroom.created_by]}</td>
+              <td onClick={() => handleDelete(chatroom.id)}>delete</td>
```

Comment:
When delete button will be clicked, show a loading spinner instead of delete icon for the respective column until it gets deleted.